Проверяем макет по критериям

ARybkova opened the pull request:
---
:mortar_board: [Проверяем макет по критериям](https://up.htmlacademy.ru/htmlcss/34/user/1947741/tasks/12)

:boom: https://htmlacademy-htmlcss.github.io/1947741-gllacy-34/10/

ARybkova committed:
Добавляет мелкие изменения

ARybkova committed:
Добавляет стилизацию index при перелистывании слайдера

ARybkova committed:
Добавляет изменения в соответствии с критериями

ARybkova committed:
Добавляет мелкие изменения

ARybkova committed:
Убирает лишние обертки, классы

avilona reviewed (commented):
1. https://disk.yandex.ru/i/jTG7oVpk8VM53w - на главной для сетки товаров надо добавить правку по переполнению
2. https://disk.yandex.ru/i/bRG_FtbWZ2cvqQ - и в блоке с преимуществами тоже, там вообще можно сделать один \<ul>, без вложенных дочерних списковэ
3. https://disk.yandex.ru/i/aPzQVMnHfEWj3A - там где блог и подписка при переполнении уезжают фоны
4. https://disk.yandex.ru/i/W3u_ZNV33ph2_w - здесь тоже надо продумать по переполнению
5. Есть доп.критерий на наличие фавиконки - https://up.htmlacademy.ru/htmlcss/34/criteries#HTML-08,  это можно быстро сделать, так что советую добавить )

ARybkova committed:
Добавляет изменения на переполнение

Переделан пункт про преимущества в index.
Учтено переполнение в фильтрах каталога.

ARybkova committed:
Добавляет мелкие правки по критериям
